<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Aleksandar" failures="2" tests="7" name="ValidateLoginTests" time="0.14" errors="0" timestamp="2021-09-15T21:17:08 CEST">
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.052"/>
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.012"/>
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.014"/>
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.018"/>
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.015"/>
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [417] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [417] but found [200]
at local.HLVTestApp.APITests.API_ValidateLoginTests.API_ValidateLogin(API_ValidateLoginTests.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- API_ValidateLogin -->
  <testcase classname="local.HLVTestApp.APITests.API_ValidateLoginTests" name="API_ValidateLogin" time="0.138">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at local.HLVTestApp.listeners.TestListener.onTestFailure(TestListener.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- API_ValidateLogin -->
</testsuite> <!-- ValidateLoginTests -->
